common/models/templates/src/trie-model.ts

```diff
@@ -24,535 +26,525 @@
+ * some languages that use the Latin script. As of 2020-04-08, only
+ * 4 out of 11 (36%) of published language models use the Latin script,
+ * so this might not actually be a great default.
+ *
+ * This uses String.prototype.normalize() to convert normalize into NFD.
+ * NFD is an easy way to separate a Latin character from its diacritics;
+ * Even then, some Latin-based orthographies use code points that,
+ * under NFD normalization, do NOT decompose into an ASCII letter and a
+ * combining diacritical mark (e.g., SENĆOŦEN).
+ *
+ * Use this only in early iterations of the model. For a production lexical
+ * model, you SHOULD write/generate your own key function, tailored to your
+ * language.
+ */
+function defaultSearchTermToKey(wordform: string): SearchKey {
   /**
-   * Converts wordforms into an indexable form. It does this by
-   * normalizing into NFD, removing diacritics, and then converting
-   * the result to lowercase.
-   *
-   * This is a very naïve implementation, that I only think will work on
-   * some languages that use the Latin script. As of 2020-04-08, only
-   * 4 out of 11 (36%) of published language models use the Latin script,
-   * so this might not actually be a great default.
+   * N.B.: this is (slightly) DIFFERENT than the version in
+   * keymanapp/lexical-model-compiler/build-trie
+   * as this is for compatibility for models built
+   * BEFORE the searchTermToKey function was bundled with
+   * all models.
    *
-   * This uses String.prototype.normalize() to convert normalize into NFD.
-   * NFD is an easy way to separate a Latin character from its diacritics;
-   * Even then, some Latin-based orthographies use code points that,
-   * under NFD normalization, do NOT decompose into an ASCII letter and a
-   * combining diacritical mark (e.g., SENĆOŦEN).
-   *
-   * Use this only in early iterations of the model. For a production lexical
-   * model, you SHOULD write/generate your own key function, tailored to your
-   * language.
+   * This compatibility version lowercases AFTER removing diacritics;
+   * the new version (bundled in future models) lowercases,
+   * NFD normalizes, THEN removes diacritics.
    */
-  function defaultSearchTermToKey(wordform: string): SearchKey {
-    /**
-     * N.B.: this is (slightly) DIFFERENT than the version in
-     * keymanapp/lexical-model-compiler/build-trie
-     * as this is for compatibility for models built
-     * BEFORE the searchTermToKey function was bundled with
-     * all models.
-     *
-     * This compatibility version lowercases AFTER removing diacritics;
-     * the new version (bundled in future models) lowercases,
-     * NFD normalizes, THEN removes diacritics.
-     */
-    return wordform
-      .normalize('NFD')
-      // Remove all combining diacritics (if input is in NFD)
-      // common to Latin-orthographies.
-      .replace(/[\u0300-\u036f]/g, '')
-      .toLowerCase() as SearchKey;
-  }
-
-  function getDefaultWordBreaker() {
```

Comment:
Does eliminating this reduce functionality in any way?

Reply:
It's not eliminated - this is handled via `import` now.  (See line 32.)

Reply:
Yeah, just this function had more than one way to obtain the default?

Reply:
I _think_ it's because one way found them when compiled into `lm-worker` (where the namespace already exists and we don't want duplication) while the other linked them when testing separately from the `lm-worker` that would establish the namespace?  

(`wordBreakers` was a top-level variable name as a compiled namespace.)

Meanwhile, `import` makes this simpler.